L1Trigger/Phase2L1ParticleFlow/interface/Region.h

```diff
@@ -61,7 +61,9 @@ namespace l1tpf_impl {
         caloOverflow = 0; emcaloOverflow = 0; trackOverflow = 0; muonOverflow = 0; pfOverflow = 0; puppiOverflow = 0;
     }
 
-    void inputSort() ;
+    void inputSort(bool doSort) ;
+
+    void outputSort(bool doSort) ;
```

Comment:
I think it would be better to rename them as inputCrop and outputCrop, since they may not sort (depending on the argument) but they always crop.